packages/core/src/lib/base_protocol.ts

```diff
@@ -97,7 +98,9 @@ export class BaseProtocol implements IBaseProtocol {
     return filterPeers(allPeersForProtocol, numPeers, maxBootstrapPeers);
   }
 
-  initializePubsubTopic(shardInfo?: ShardInfo): PubsubTopic[] {
+  initializePubsubTopic(
+    shardInfo?: ShardInfo | ContentTopicInfo
```

Comment:
`ShardInfo | ContentTopicInfo` this is a common pattern at this point - makes sense to have a type alias

Reply:
added in latest commit